projects/storefrontlib/cms-components/navigation/search-box/search-box.component.html

```diff
@@ -386,19 +386,37 @@ <h3 class="products-header" *ngIf="result.products?.length">
   <span id="initialDescription" class="cx-visually-hidden">
     {{ 'searchBox.initialDescription' | cxTranslate }}
   </span>
-  <div
-    *ngIf="result.suggestions?.length || result.products?.length"
-    aria-live="assertive"
-    class="cx-visually-hidden"
-  >
-    {{
-      'searchBox.suggestionsResult'
-        | cxTranslate: { count: result.suggestions?.length }
-    }}
-    {{
-      'searchBox.productsResult'
-        | cxTranslate: { count: result.products?.length }
-    }}
-    {{ 'searchBox.initialDescription' | cxTranslate }}
-  </div>
+  <ng-container *cxFeature="'a11ySearchboxAssistiveMessage'">
+    <div
+      *ngIf="result.suggestions?.length || result.products?.length"
+      aria-live="assertive"
+      class="cx-visually-hidden"
+    >
+      {{
+        'searchBox.suggestionsResult'
+          | cxTranslate: { count: result.suggestions?.length }
+      }}
+      {{
+        'searchBox.productsResult'
+          | cxTranslate: { count: result.products?.length }
+      }}
+    </div>
+  </ng-container>
+  <ng-container *cxFeature="'!a11ySearchboxAssistiveMessage'">
+    <div
+      *ngIf="result.suggestions?.length || result.products?.length"
+      aria-live="assertive"
+      class="cx-visually-hidden"
+    >
+      {{
+        'searchBox.suggestionsResult'
+          | cxTranslate: { count: result.suggestions?.length }
+      }}
+      {{
+        'searchBox.productsResult'
+          | cxTranslate: { count: result.products?.length }
+      }}
+      {{ 'searchBox.initialDescription' | cxTranslate }}
```

Comment:
Could we just have `<ng-container *cxFeature="'a11ySearchboxAssistiveMessage'">` wrapping `initialDescription` instead?